llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -15,6 +15,15 @@
 namespace llvm {
 namespace memprof {
 
+// The versions of the indexed MemProf format
+enum MemProfVersion {
```

Comment:
Perhaps call this IndexedVersion so when we spell it out with the namespace (e.g. `memprof::IndexedVersion`) it's clear to the reader which format we are referring to?